src/ecalc/cli/tests/test_app.py

```diff
@@ -526,6 +526,25 @@ def test_csv_resampled(self, simple_run, monkeypatch, snapshot):
         output_text = result.stdout
         snapshot.assert_match(output_text, snapshot_name="results_resampled.csv")
 
+    @pytest.mark.snapshot
+    def test_json_resampled(self, simple_run, monkeypatch, snapshot):
```

Comment:
Is a snapshot test needed here? Could it be tested without adding more json snapshots that need to be updated?

Reply:
as mentioned, I think so. But I can and should specify why in this test instead of just adding it